src/libertem/web/cli.py

```diff
@@ -1,20 +1,28 @@
 import click
+import logging
+
+log_values = "Allowed values are 'critical', 'error', 'warning', 'info', 'debug'."
```

Comment:
Adding the available levels in help was a bit lengthy, also it can be reused with the error msg.